apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -17,7 +17,7 @@ import {VideoPlayer} from '@primer/react-brand'
 
 ```jsx live
 <VideoPlayer title="GitHub media player">
-  <VideoPlayer.Source src="/brand/assets/example.mp4" />
+  <VideoPlayer.Source src="https://github.com/primer/brand/raw/refs/heads/main/apps/storybook/static/example.mp4" />
```

Comment:
In hindsight this feels a little risky for a number of reasons....

Could we instead link directly to the origin URL, which should fix the issue by not appending a trailing slash.

```suggestion
  <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" />
```

Reply:
Oh cool, hadn't realised an absolute URL would fix it. Great suggestion, thanks!